core/src/main/java/org/jobrunr/dashboard/ui/model/problems/CarbonIntensityApiErrorProblemHandler.java

```diff
@@ -36,7 +36,7 @@ public void onChange(List<JobRunrMetadata> metadataList) {
         if (this.metadata == null || this.metadata.size() != metadataList.size()) {
             dismiss();
             if (!metadataList.isEmpty()) {
-                problems.addProblem(new CarbonIntensityApiErrorProblem(InstantUtils.max(metadataList.stream().map(JobRunrMetadata::getCreatedAt))));
+                problems.addProblem(new CarbonIntensityApiErrorProblem(metadataList, InstantUtils.max(metadataList.stream().map(JobRunrMetadata::getCreatedAt))));
```

Comment:
Since we now pass `metadataList` to  `CarbonIntensityApiErrorProblem`, we can remove it's second constructor and compute `InstantUtils.max(metadataList.stream().map(JobRunrMetadata::getCreatedAt))` there. Similarly to what we do for the other problems.